src/login/register.jsx

```diff
@@ -13,21 +13,21 @@ const Register = () => {
 
   const { userLoggedIn } = useAuth();
 
-  const onSubmit = async (e) => {
-    e.preventDefault();
-    if (!isRegistering) {
```

Comment:
Is the idea to use `isRegistering` for a loading animation as well?

Reply:
yes!